src/mmda/parsers/pdfplumber_parser.py

```diff
@@ -1,15 +1,56 @@
-from typing import Optional, List, Dict, Union
-
+import itertools
 import string
+from typing import Dict, List, Optional, Union
+
 import pdfplumber
+import pdfplumber.utils as ppu
+from pdfplumber._typing import T_num
 
-import itertools
-from mmda.types.span import Span
-from mmda.types.box import Box
+from mmda.parsers.parser import Parser
 from mmda.types.annotation import SpanGroup
+from mmda.types.box import Box
 from mmda.types.document import Document
-from mmda.parsers.parser import Parser
+from mmda.types.metadata import Metadata
 from mmda.types.names import PagesField, RowsField, SymbolsField, TokensField
+from mmda.types.span import Span
+
+
+class WordExtractorWithFontInfo(ppu.WordExtractor):
+    """Override WordExtractor methods to append additional char-level info."""
+
+    def __init__(
+        self,
+        x_tolerance: T_num = ppu.DEFAULT_X_TOLERANCE,
+        y_tolerance: T_num = ppu.DEFAULT_Y_TOLERANCE,
+        keep_blank_chars: bool = False,
+        use_text_flow: bool = False,
+        horizontal_ltr: bool = True,
+        vertical_ttb: bool = True,
+        extra_attrs: Optional[List[str]] = None,
+        split_at_punctuation: Union[bool, str] = False,
+        append_attrs: Optional[List[str]] = None,
+    ):
+        super().__init__(
+            x_tolerance,
+            y_tolerance,
+            keep_blank_chars,
+            use_text_flow,
+            horizontal_ltr,
+            vertical_ttb,
+            extra_attrs,
+            split_at_punctuation,
+        )
```

Comment:
```suggestion
        super().__init__(
            x_tolerance=x_tolerance,
            y_tolerance=y_tolerance,
            keep_blank_chars=keep_blank_chars,
            use_text_flow=use_text_flow,
            horizontal_ltr=horizontal_ltr,
            vertical_ttb=vertical_ttb,
            extra_attrs=extra_attrs,
            split_at_punctuation=split_at_punctuation,
        )
```

I would switch to kwargs in case the order changes in pdfplumber.